louis/models/openai.py

```diff
@@ -15,20 +15,21 @@ def safe_get(key):
     return value
 
 OPENAI_API_KEY = safe_get("OPENAI_API_KEY")
-AZURE_OPENAI_SERVICE = safe_get("AZURE_OPENAI_SERVICE")
+AZURE_OPENAI_ENDPOINT = safe_get("AZURE_OPENAI_ENDPOINT")
+OPENAI_API_ENGINE = safe_get('OPENAI_API_ENGINE')
 
 openai.api_type = "azure"
 openai.api_key = OPENAI_API_KEY
-openai.api_base = f"https://{AZURE_OPENAI_SERVICE}.openai.azure.com"
+openai.api_base = f"{AZURE_OPENAI_ENDPOINT}"
 openai.api_version = "2023-05-15"
 
 enc = tiktoken.get_encoding("cl100k_base")
 
 def fetch_embedding(tokens):
     """Fetch embedding for a list of tokens from the Microsoft Azure OpenAI API"""
     response = openai.Embedding.create(
-        input=tokens,
-        engine="ada"
+        input = tokens,
+        engine = OPENAI_API_ENGINE
```

Comment:
naming of your environment is inconsistent: you have AZURE_OPENAI_ENDPOINT and OPENAI_API_ENGINE. Since both those parameters feed to the openai library and are independent of Azure, let's call it OPENAI_ENDPOINT and OPENAI_API_ENGINE